codes/quantum/oscillators/Niset-Andersen-Cerf.yml

```diff
@@ -0,0 +1,37 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: niset_andersen_cerf
+physical: oscillators
+logical: oscillators
+
+
+name: 'Niest-Andersen-Cerf Code'
+
+introduced: '\cite{arXiv:0710.4858}'
+
+description: 'Encodes two photon coherent states (or a general squeezed state) into four photons. There are two variations, a deterministic protocol that offers recovery against a single photon error, or a probabalistic that can protect against multiple errors with post selection.'

```

Comment:
Encodes coherent states \(|\alpha\rangle|\beta|\rangle\) over two modes into four modes such that the values \(\alpha,\beta\) are recoverable after a single mode erasure. There are two variations, a deterministic protocol that offers recovery against a single mode erasure, or a probabalistic that can protect against multiple errors with post selection. This code is effectively protecting classical information stored in \(\alpha,\beta\) using quantum operations.